relayer/src/keyring.rs

```diff
@@ -138,7 +138,7 @@ impl KeyStore for Memory {
     }
 
     fn remove_key(&mut self, key_name: &str) -> Result<(), Error> {
-        self.keys.remove(key_name).ok_or(Error::key_not_found())?;
+        self.keys.remove(key_name).ok_or(Error::key_not_found)?;
```

Comment:
It seems like it was correct before.

Reply:
It wasn't but the fix suggested by the compiler is even worse. Will fix it manually.

Reply:
My bad, I actually misapplied the compiler suggestion.

Reply:
Fixed

Reply:
Fixed